base_app/restaurant.py

```diff
@@ -10,6 +10,7 @@ def __init__(self, id, name, categories, location, phone, display_phone, rating,
         self.display_phone = display_phone
         self.rating = rating
         self.price = price
+        self.is_closed = is_closed
```

Comment:
Definitely makes sense to grab only restaurants that are open - is there a reason you went with closed instead of open_now? Is it not available as a param on this endpoint?

Reply:
`is_closed` is the value specified in the Yelp API return object. I simply kept it the same. It doesn't matter to me, if you'd like we can change it.